core/templates/dev/head/pages/base.html

```diff
@@ -265,17 +265,18 @@
                               </div>
                             </li>
 
-                            <li class="dropdown oppia-navbar-clickable-dropdown pull-right">
+                            <li class="dropdown oppia-navbar-clickable-dropdown pull-right protractor-test-about-oppia-list-item">
                               <a class="oppia-navbar-tab">
                                 <span translate="I18N_TOPNAV_ABOUT"></span>
                                 <span class="caret"></span>
                               </a>
                               <ul class="dropdown-menu oppia-navbar-dropdown" ng-mouseover="onMouseoverDropdownMenu($event)" ng-mouseleave="onMouseoutDropdownMenu($event)">
-                                <li><a href="/about" translate="I18N_TOPNAV_ABOUT_OPPIA"></a></li>
-                                <li><a href="/teach" translate="I18N_TOPNAV_TEACH_WITH_OPPIA"></a></li>
+                                <li><a class="protractor-test-about-link"
+                                href="/about" translate="I18N_TOPNAV_ABOUT_OPPIA"></a></li>
```

Comment:
Align the leftmost character of href with the leftmost character of class -- or don't break the line at all. This is the convention we use in this codebase when indenting html.